src/Akeneo/Connectivity/Connection/back/Application/Webhook/Validation/ExternalUrlValidator.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Application\Webhook\Validation;
+
+use Akeneo\Connectivity\Connection\Application\Webhook\Service\DnsLookupInterface;
+use Symfony\Component\HttpFoundation\IpUtils;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ExternalUrlValidator extends ConstraintValidator
+{
+    private const DOMAIN_BLACKLIST = [
+        'localhost',
+        'elasticsearch',
+        'memcached',
```

Comment:
Should we also add object-storage & mysql ?

```suggestion
        'memcached',
        'object-storage',
        'mysql',
```

Reply:
I've added the question to the ticket, waiting for clarification.